### PR TITLE
Postman Tests

### PR DESCRIPTION
**Pull request by Pantani**
- [x] tx endpoint
- [x] staking endpoint
- [x] token endpoint
- [x] domain endpoint
- [x] collection endpoint
- [x] market data
- [x] observers
- [x] POST batch requests

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -118,6 +120,52 @@ golint: go-lint
...
+ifeq (,$(shell which newman))
+	@echo "  >  Installing Postman Newman"
+	@-brew install newman
+endif
+
+## run-newman: Run Postman Newman tests.
+run-newman: install-newman newman-txs newman-tokens newman-staking newman-collections newman-domains newman-observer newman-healthcheck
+
+## newman-txs: Run Newman tests for Transaction API's.
+newman-txs:
```
Comment (vikmeup) [inline Makefile]: lots of code duplication. 

Did you consider improving interface to run tets?

`make integrations tokens`
`make integrations staking`

 ? or anything similar

It's pointless to create each command for specifying each file

File: pkg/tests/postman/collection_data.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[
```
Comment (vikmeup) [inline pkg/tests/postman/collection_data.json]: any reason to add `_data` to each file?

Comment (Pantani) [reply pkg/tests/postman/collection_data.json]: Any object of the JSON array is an iteration the postman going to do with this API, if a new platform starts to implement staking, we need to update the `staking_data.json` file

https://learning.postman.com/docs/postman/collec
tion-runs/running-multiple-iterations/

**Comment by vikmeup** [comment]
I see:
```
Iteration 1/58
  GET https://***/v1/harmony/one1e4mr7tp0a76wnhv9xd0wzentdjnjnsh3fwzgfv
```

Can we test v2 for transaction ? v1 is oudated already, we can probably remove it already?

**Pull request merged**